Project.toml

```
@@ -76,6 +78,7 @@ OffsetArrays = "1.12"
 P4est = "0.4.9"
 Polyester = "0.7.5"
 PrecompileTools = "1.1"
+Preferences = "1"
```

```suggestion
Preferences = "1.3"
```
SciMLBase.jl is restricted to Preferences.jl v1.3 or higher.

---

Thanks!